packages/cloudflare/src/cli/build/patches/ast/util.ts

```diff
@@ -17,9 +19,9 @@ export type RuleConfig = NapiConfig & { fix?: string };
  * @param rule The rule. Either a yaml string or an instance of `RuleConfig`
  * @param root The root node
  * @param once only apply once
- * @returns A list of edits.
+ * @returns A list of edits and a list of matches.
  */
-export function getRuleEdits(rule: string | RuleConfig, root: SgNode, { once = false } = {}) {
+export function applyRule(rule: string | RuleConfig, root: SgNode, { once = false } = {}) {
```

Comment:
The `patchCode` code comment needs to also be updated:
https://github.com/opennextjs/opennextjs-cloudflare/blob/a7fb792950bbf70435a6e141d8fe8863250ecd3d/packages/cloudflare/src/cli/build/patches/ast/util.ts#L73

```diff
- * use `getRuleEdits` to apply multiple rules.
+  * uses `getRuleEdits` to apply multiple rules.
```

Reply:
I think "use" is what we want here